test/integration/licence_test.rb

```diff
@@ -220,7 +220,7 @@ class LicenceTest < ActionDispatch::IntegrationTest
 
         should "add GA4 information click attributes" do
           data_module = page.find("article")["data-module"]
-          expected_data_module = "ga4-auto-tracker ga4-link-tracker"
+          expected_data_module = "ga4-link-tracker ga4-auto-tracker"
```

Comment:
Quick question: would we also want to test for instances where `ga4-auto-tracker` isn't supposed to appear?

Reply:
Good idea, done :+1: